build-conventions-secant/src/main/kotlin/secant.android-build-conventions.gradle.kts

```diff
@@ -52,6 +52,28 @@ pluginManager.withPlugin("com.android.library") {
     }
 }
 
+pluginManager.withPlugin("com.android.test") {
+    project.the<com.android.build.gradle.TestExtension>().apply {
+        configureBaseExtension(isLibrary = true)
+
+        defaultConfig {
+            minSdk = project.property("ANDROID_LIB_MIN_SDK_VERSION").toString().toInt()
+            targetSdk = project.property("ANDROID_TARGET_SDK_VERSION").toString().toInt()
```

Comment:
No, the definitions are different for these reasons:
- The one in `secant.android-build-conventions.gradle.kts` is definition of the `com.android.test` type of Android plugin and this type of plugin is according to documentation derived from `com.android.library` module, so that's why I used here the same min SDK version
- And the definition for emulator.wtf in `secant.emulator-wtf-conventions.gradle.kts` use the same as the `app` module, because the the emulator.wtf service requires it (see failed [tests results](https://results.emulator.wtf/?run=f67aa6e1-92a5-44e2-a380-448d23c24ab9&key=QxlwXmkkDq3W-v9zFom0XtOWjwOwx46A94mVAioq) on emulator.wtf with min version same as in library modules). And below is result from error-logcat.txt downloaded from the service:
```
--------- beginning of system
10-12 10:02:13.999  1630  1656 W PackageManager: Failed parse during installPackageLI
10-12 10:02:13.999  1630  1656 W PackageManager: android.content.pm.PackageParser$PackageParserException: /data/app/vmdl651849006.tmp/base.apk (at Binary XML file line #7): Requires newer sdk version #27 (current version is #24)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at android.content.pm.PackageParser.parseBaseApk(PackageParser.java:934)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at android.content.pm.PackageParser.parseClusterPackage(PackageParser.java:830)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at android.content.pm.PackageParser.parsePackage(PackageParser.java:794)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at com.android.server.pm.PackageManagerService.installPackageLI(PackageManagerService.java:14883)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at com.android.server.pm.PackageManagerService.installPackageTracedLI(PackageManagerService.java:14830)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at com.android.server.pm.PackageManagerService.-wrap25(PackageManagerService.java)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at com.android.server.pm.PackageManagerService$9.run(PackageManagerService.java:12324)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at android.os.Handler.handleCallback(Handler.java:751)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at android.os.Handler.dispatchMessage(Handler.java:95)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at android.os.Looper.loop(Looper.java:154)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at android.os.HandlerThread.run(HandlerThread.java:61)
10-12 10:02:13.999  1630  1656 W PackageManager: 	at com.android.server.ServiceThread.run(ServiceThread.java:46)
10-12 10:02:14.001  1397  1397 E         : Couldn't opendir /data/app/vmdl651849006.tmp: No such file or directory
10-12 10:02:14.003  4473  4473 I art     : System.exit called, status: 1
10-12 10:02:14.003  4473  4473 I AndroidRuntime: VM exiting with result code 1.
```
But it can be just a problem of the service and we can ask the maintainer for analysing it or we can simply increase the min SDK version for the whole test module. What do you think? Thank you.